Sources/TokamakStaticHTML/Views/Text/Text.swift

```diff
@@ -158,6 +158,35 @@ extension Text: AnyHTML {
   }
 }
 
+@_spi(TokamakStaticHTML) extension Text: HTMLConvertible {
+  @_spi(TokamakStaticHTML) public var innerHTML: String? {
+    let proxy = _TextProxy(self)
+    let innerHTML: String
+    switch proxy.storage {
+    case let .verbatim(text):
+      innerHTML = proxy.environment.domTextSanitizer(text)
+    case let .segmentedText(segments):
+      innerHTML = segments
+        .map {
+          TextSpan(
+            content: proxy.environment.domTextSanitizer($0.0.rawText),
+            attributes: Self.attributes(
+              from: $0.1,
```

Comment:
nit: can we have labels added to this tuple type for clarity?
```suggestion
            content: proxy.environment.domTextSanitizer($0.storage.rawText),
            attributes: Self.attributes(
              from: $0.modifier,
```